### ISSUE TITLE
How to finetune the infinity based on your provided weights?

### ISSUE THREAD

**Issue by xuanyuzhang21** [FoundationVision/Infinity#29]
Thanks for your great work and code release!  Could you provide any instructions for finetuning infinity based on your provided checkpoints?

**Comment by JeyesHan** [comment]
@xuanyuzhang21 Thanks for your appreciation to Infinity. Finetuning is very simple. You can run [train.sh](https://github.com/FoundationVision/Infinity/blob/main/scripts/train.sh) with --rush_resume [infinity_2B.pth].

**Comment by hakanmuluk** [comment]
Firstly, thanks for the great work! However, there happens lots of shape mismatches when I directly use --rush_resume with the model you have provided, is there a solution for it?

**Comment by JeyesHan** [comment]
@hakanmuluk Have you ever solved the problems? If not, could you pull latest commit and try again?

**Comment by Kaiwen-Zhu** [comment]
@JeyesHan A minor typo in README: `--rush_resume=[infinity_vae_d32reg.pth]` -> `--rush_resume=[infinity_2b_reg.pth]`

**Comment by hakanmuluk** [comment]
@JeyesHan No, I could not solve them. I tried with the latest commit and the shape mismatches looks disappeared but I still get meaningless images. However, the loss and accuracy starts in a better region suggesting that the model is loaded without any problem. I think that model is finetuned but there is something incompatible with the training and infer.sh so I cannot get good images in the inference but get images similar to the one shared in the other issue about finetuning. Is the infer.sh compatible with the trained version of the 2b model you have provided? Also, there are some problems in the code: visualizer is not defined in train.py, there should be something like enable_model_cache in the interactive_ipynb.

**Comment by metildachee** [comment]
Hello @xuanyuzhang21 if its possible could you please share how much GPU was necessary for your training set? I tried using `rush_resume` on `2b` and `vae_ckpt` on `32b` but 40GPU wasn't sufficient. Did you use more than that? Looking forward! Thank you very much! 🙏🏻

**Comment by xuanyuzhang21** [comment]
@metildachee I finetuned the model via a single 80GB a100.

**Comment by JeyesHan** [comment]
@hakanmuluk Can you post your b1_stdout.txt and interactive_ipynb file here. Such that I can find the problem.

**Comment by metildachee** [comment]
Hi authors and fellow coders! My results are disappointing after 16 epochs, the SFT model can't generate reasonable images qwq, any advice on how many epochs to finetune on? Thanks so much in advice! 😊
